packages/polymath-issuer/src/pages/ticker/components/ReserveTickerForm.js

```diff
@@ -60,6 +60,12 @@ export const ReserveTickerFormComponent = ({ handleSubmit }) => (
         <FormItem.Input
           component={TextInput}
           placeholder="Up to 10 characters (example: TORO-A)"
+          onChange={e => {
+            let value = e.target.value || '';
+            value = value.toUpperCase().trim();
+            // setFieldValue(field.name, value);
+            e.target.value = value;
```

Comment:
I think it's less bug-prone to do this in CSS instead with `text-transform: uppercase` for the display of the input. And do like you do with `.toUpperCase()` on submit.